src/13_file_io.py

```diff
@@ -8,11 +8,24 @@
 # Open up the "foo.txt" file (which already exists) for reading
 # Print all the contents of the file, then close the file
 
-# YOUR CODE HERE
+text = open('foo.txt')
+print(text.read())
+text.close()
 
 # Open up a file called "bar.txt" (which doesn't exist yet) for
 # writing. Write three lines of arbitrary content to that file,
 # then close the file. Open up "bar.txt" and inspect it to make 
 # sure that it contains what you expect it to contain
+print('\n\n')
+    
+new_file = open('bar.txt', 'w')
+new_file.write("I went to have an interview as a carpenter today")
```

Comment:
You can also do `new_file.write("I went to have an interview as a carpenter today. \n add another line \n add another line.)`
`\n` is the code for a new line.